### ISSUE TITLE
CPU Power missing from API

### ISSUE THREAD

**Issue by patrick-robin** [argotronic/argus_data_api#3]
Is it possible to surface the CPU power through the API. Unless I'm missing something it doesn't appear as one of the valid sensor types?

**Comment by argotronic** [comment]
Yes, CPU power is indeed still missing from the list of sources. I have it on my list to add in the near future (no further modifications are required by client code other than recompiling with the new header as this will be a new `ARGUS_MONITOR_SENSOR_TYPE` I'm afraid.

**Comment by patrick-robin** [comment]
> Yes, CPU power is indeed still missing from the list of sources. I have it on my list to add in the near future (no further modifications are required by client code other than recompiling with the new header as this will be a new `ARGUS_MONITOR_SENSOR_TYPE` I'm afraid.

No worries, thanks for the update :)
